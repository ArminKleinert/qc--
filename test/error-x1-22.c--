
register bits8 c;

section "data" 
{
    x:
    bits32[4]; 
}

foo () {
    bits32 [x] = foo();        
}







