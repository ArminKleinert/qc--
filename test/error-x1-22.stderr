error-x1-22.c--:21.1 Error: lvalue must be a register
